src/spec/color-spec/base-color-set.ts

```diff
@@ -57,5 +57,17 @@ export const BaseColorSet: ColorSet = {
             Border: "#DD6666",
             Background: "#FDF7F7", // Validation Background
         },
+        Blue: {
```

Comment:
You are adding into the `base-color-set` which is only for LifeSG and Legacy. You might want to add it to `bookingsg-color-set` too?